packages/orbit-components/package.json

```diff
@@ -124,12 +124,12 @@
     "dotenv-safe": "^8.2.0",
     "fs-extra": "^9.0.1",
     "glob": "^7.1.2",
-    "globby": "^11.0.4",
     "isomorphic-unfetch": "^3.1.0",
     "jest-matchmedia-mock": "^1.1.0",
     "jest-styled-components": "^7.0.5",
     "jscodeshift": "^0.11.0",
     "jsdom": "^16.5.3",
+    "lodash": "^4.17.21",
```

Comment:
i think we should add also `babel-plugin-lodash` then

Reply:
Why? 🤔 `lodash` is used only for sorting in `.size-limit.js`.

Reply:
ah, right, so in this case i think it will be better to just have own functions and do not add dependency. Like take it from [here](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_sortby-and-_orderby), for instance